tile_generator/templates/jobs/spec

```diff
@@ -112,9 +112,11 @@ properties:
 
 consumes:
 {% for release in context.releases.values() if release.consumes %}
-{% for link_type, link in release.consumes.iteritems() %}
-  - name: {{ link.name or link_type }}
-    type: {{ link_type }}
-    optional: {{ link.optional or true }}
+{% for link_name, link in release.consumes.iteritems() %}
+  - name: {{ link_name }}
+    type: {{ link.type or link_name }}
+{% if link.optional %}
```

Comment:
We don't want to explicitly have a `optional: false` ever?

Reply:
Setting optional to 'false' is a redundant statement, the act of adding an entry under "consumes" in the spec means it is required, the only use of 'optional' in the bosh documentation is when trying to set it to true.
I am just trying to keep the same defaults and outputs matching the documentation I have seen and confirmed in my testing.